geometry_msgs/Point min
geometry_msgs/Point max
float32 occupancy
float32 observation_count
bool densify
bool is_observation_certainty
---
bool success
